.pre-commit-config.yaml

```diff
@@ -4,7 +4,7 @@ ci:
 
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.6.0
+    rev: v5.0.0
```

Comment:
@nstarman , do you know if this accepts hash and still auto-update like https://github.com/astropy/astropy/pull/17082 ? Anyway, just curious. Not related to this PR.

Reply:
I don't believe it does (yet?).